pkg/observer/observer_test_helper.go

```diff
@@ -398,15 +398,15 @@ func loadExporter(t *testing.T, obs *Observer, opts *testExporterOptions, oo *te
 	return nil
 }
 
-func loadObserver(t *testing.T, obs *Observer, base *sensors.Sensor, sens []*sensors.Sensor, notestfail bool) error {
-	if err := base.Load(context.TODO(), obs.bpfDir, obs.mapDir, obs.ciliumDir); err != nil {
+func loadObserver(t *testing.T, base *sensors.Sensor, sens []*sensors.Sensor, notestfail bool) error {
```

Comment:
Could we rename this to `loadSensors` since it no longer has anything to do with the observer?

Reply:
Yes but I'll do it in follow up.